data-model/annotations/schema/annotation.json

```diff
@@ -2,99 +2,207 @@
   "$schema": "http://json-schema.org/draft-2020-12/schema#",
   "type": "object",
   "properties": {
-    "ods:id": { "type": "string" },
-    "ods:type": { "const": "Annotation" },
```

Comment:
this "type" is rdf:type based on w3c https://www.w3.org/ns/anno.jsonld. The other "types" are FDO Types so should ods.

Reply:
so oa:type?

Reply:
No. rdf:type. anno.json refers to the rdf name space in the json ld context. https://www.w3.org/1999/02/22-rdf-syntax-ns#